integration/src/test/java/org/kie/hacep/PodAsLeaderSnapshotTest.java

```diff
@@ -26,12 +26,16 @@
 import org.kie.hacep.core.infra.election.State;
 import org.kie.hacep.message.SnapshotMessage;
 import org.kie.remote.RemoteKieSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static org.junit.Assert.*;
 import static org.kie.remote.util.SerializationUtil.deserialize;
 
 public class PodAsLeaderSnapshotTest extends KafkaFullTopicsTests{
 
+    private Logger logger = LoggerFactory.getLogger("org.hacep");
```

Comment:
This is the reason https://github.com/kiegroup/openshift-drools-hacep/blob/master/integration/src/test/resources/logback-test.xml#L31

Reply:
Again, it seems unusual to me to have logger name which does not correspond to package in which our classes are. If we use correct package name "org.kie.hacep" instead of "org.hacep" then my approach would work.